src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: 10px;
+  width: $calendar-width;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $day-size;
```

Comment:
The variable `$day-size` is used here, but it's not defined within this file. Ensure that this variable is defined in a shared variables file or within this file to avoid compilation errors.